site/assets/scss/_component-examples.scss

```diff
@@ -361,7 +363,7 @@
 
   pre code {
     @include font-size(inherit);
-    color: $gray-900; // Effectively the base text color
+    color: var(--bs-body-color); // Effectively the base text color
     word-wrap: normal;
   }
 }
```

Comment:
Missing changes on highlight-toolbar.

Reply:
Updated the comments via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/dcb058594832bec1fcb9dfa81e2d8c2cd50b142c. The addition of `.btn-clipboard` is not needed because of https://github.com/twbs/bootstrap/pull/37817/files (I'll remove the comment then when this commit will come back to Boosted)